test/fixtures/default/invalid.css

```diff
@@ -276,4 +278,4 @@ unknown {
 .last {
 	color: #fff;
 	/* stylelint-disable-next-line no-missing-end-of-source-newline */
-}
\ No newline at end of file
+}
```

Comment:
Your IDE has inserted a newline at the end here. It needs to not exist to assert the linter is working.

Reply:
Heh, that's what I get for using GitHub's built-in editing tools. Will have to download it locally to fix this. 🙈